<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Medium Scale Combat for 5e Datasheet Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGroupBox" name="statBlockWidget_2">
        <property name="title">
         <string>Stat Block</string>
        </property>
        <layout class="QVBoxLayout" name="statBlockWidget">
         <item>
          <layout class="QFormLayout" name="baseStatsWidget">
           <item row="0" column="0">
            <widget class="QLabel" name="nameLabel">
             <property name="text">
              <string>&amp;Name</string>
             </property>
             <property name="buddy">
              <cstring>nameInput</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="nameInput">
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>128</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string/>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="hpLabel">
             <property name="text">
              <string>Average &amp;hit points</string>
             </property>
             <property name="buddy">
              <cstring>hpBox</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="hpBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="acLabel">
             <property name="text">
              <string>&amp;Armor class</string>
             </property>
             <property name="buddy">
              <cstring>acBox</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="acBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="value">
              <number>10</number>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="speedLabel">
             <property name="text">
              <string>&amp;Speed</string>
             </property>
             <property name="buddy">
              <cstring>speedBox</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="speedBox">
             <property name="suffix">
              <string> ft.</string>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>999</number>
             </property>
             <property name="value">
              <number>30</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="profLabel">
             <property name="text">
              <string>&amp;Proficiency bonus</string>
             </property>
             <property name="buddy">
              <cstring>profBox</cstring>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="profBox">
             <property name="prefix">
              <string>+</string>
             </property>
             <property name="maximum">
              <number>20</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="abilityScoresWidget">
           <property name="title">
            <string>Ability Scores</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="1">
             <widget class="QSpinBox" name="intBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="QSpinBox" name="dexBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="strBox">
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="buttonSymbols">
               <enum>QAbstractSpinBox::UpDownArrows</enum>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="intLabel">
              <property name="text">
               <string>&amp;Intelligence</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>intBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="wisLabel">
              <property name="text">
               <string>&amp;Wisdom</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>wisBox</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QSpinBox" name="conBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="dexLabel">
              <property name="text">
               <string>&amp;Dexterity</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>dexBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QSpinBox" name="wisBox">
              <property name="suffix">
               <string/>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="strLabel">
              <property name="text">
               <string>S&amp;trength</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>strBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="chaLabel">
              <property name="text">
               <string>&amp;Charisma</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>chaBox</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QSpinBox" name="chaBox">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>30</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="conLabel">
              <property name="text">
               <string>C&amp;onstitution</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="buddy">
               <cstring>conBox</cstring>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="attackLabel">
           <property name="text">
            <string>Attacks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="attacksListView">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="attackButtonsLayout">
           <item row="0" column="1">
            <widget class="QPushButton" name="addSaveAttackButton">
             <property name="text">
              <string>Add an attack based on saving throw</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="deleteAttackButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete attack</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="addRollAttackButton">
             <property name="text">
              <string>Add an attack based on attack rolls</string>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="editAttackButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit attack</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="multiattackLabel">
           <property name="text">
            <string>Multiattacks</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListView" name="multiattackListView">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="multiattackButtonsLayout">
           <item>
            <widget class="QPushButton" name="addMultiattackButton">
             <property name="text">
              <string>Add multiattack</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editMultiattackButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Edit multiattack</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="deleteMultiattackButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete multiattack</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="previewsWidget">
        <property name="tabPosition">
         <enum>QTabWidget::North</enum>
        </property>
        <property name="tabShape">
         <enum>QTabWidget::Rounded</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="elideMode">
         <enum>Qt::ElideNone</enum>
        </property>
        <property name="tabsClosable">
         <bool>false</bool>
        </property>
        <property name="tabBarAutoHide">
         <bool>false</bool>
        </property>
        <widget class="QWidget" name="srdPreviewWidget">
         <attribute name="title">
          <string>5e Stat Block Preview</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="MplCanvas" name="srdPreviewPlot" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="mediumScaleCombatPreviewWidget">
         <attribute name="title">
          <string>Medium Combat for 5e Stat Block Preview</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="MplCanvas" name="mediumScalePreviewPlot" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionExportDatasheet"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionOpen">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionExportDatasheet">
   <property name="text">
    <string>Export datasheet</string>
   </property>
   <property name="toolTip">
    <string>Export datasheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplCanvas</class>
   <extends>QWidget</extends>
   <header>custom_ui.matplotlib</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>hpBox</tabstop>
  <tabstop>acBox</tabstop>
  <tabstop>speedBox</tabstop>
  <tabstop>profBox</tabstop>
  <tabstop>strBox</tabstop>
  <tabstop>dexBox</tabstop>
  <tabstop>conBox</tabstop>
  <tabstop>intBox</tabstop>
  <tabstop>wisBox</tabstop>
  <tabstop>chaBox</tabstop>
  <tabstop>attacksListView</tabstop>
  <tabstop>addRollAttackButton</tabstop>
  <tabstop>addSaveAttackButton</tabstop>
  <tabstop>editAttackButton</tabstop>
  <tabstop>deleteAttackButton</tabstop>
  <tabstop>multiattackListView</tabstop>
  <tabstop>addMultiattackButton</tabstop>
  <tabstop>editMultiattackButton</tabstop>
  <tabstop>deleteMultiattackButton</tabstop>
  <tabstop>previewsWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
